packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx

```diff
@@ -142,65 +180,20 @@ const renderTest = async (props: Partial<ReleaseSummaryProps>) => {
   )
 }
 
-describe.skip('ReleaseSummary', () => {
-  beforeEach(async () => {
-    vi.clearAllMocks()
-
-    await renderTest({})
-  })
-
-  it('lists the release title and description', () => {
-    screen.getByText('Release title')
-    screen.getByText('Release description')
-  })
-
-  it('shows the number of documents in release', () => {
-    screen.getByText('2 documents')
-  })
-
-  it('shows the creator and date of release', () => {
-    within(screen.getByTestId('summary')).getByText('just now')
-  })
-
-  it('shows whether release has been published', () => {
-    screen.getByText('Not published')
-  })
-
-  it('shows a list of collaborators on release', () => {
-    const collaborators = getByDataUi(screen.getByTestId('summary'), 'AvatarStack')
-    expect(collaborators.childNodes).toHaveLength(2)
-  })
+describe('ReleaseSummary', () => {
+  describe('for an active release', () => {
+    beforeEach(async () => {
+      await renderTest({})
+      await vi.waitFor(() => screen.getByTestId('document-table-card'), {
+        timeout: 5000,
+        interval: 500,
```

Comment:
@jordanl17 seriously, was the issue the whole time just a timeout? 🥲

Reply:
Not quite - this seemed to be some flakiness in the tests, and I'm not quite sure why. Adding in a longer timeout seems to resolve the flake, but there were quite a few other parts necessary to actually get these tests to work:
1. awaiting the `waitFor` for the spinner to go away and the table to render
2. Updating the mock for `DefaultPreview` to pass the props through
3. Something else that I can't remember...

Reply:
God, this is so frustrating 😭 😂